src/utils.ts

```diff
@@ -0,0 +1 @@
+export const normalizeSlashes = (filePath: string) => filePath.replace(/\\/g, "/");
```

Comment:
This also looks a little strange to be alone in this file. Maybe combine this and locateConfigFile into ProjectDirHelpers.ts?

Reply:
It can't go with `locateConfigFile` since it's also required for `Emit.ts`, and importing it from `cli` directory would kind of break isolation.

`src/utils.ts` for generic utils also exists in #705, so it wouldn't be alone.